### ISSUE TITLE
Adding "Restaurant Opened for Delivery" feature in Essentials.

### ISSUE THREAD

**Issue by rohit154** [covid19india/covid19india.github.io#1945]
To make people aware of the restaurants providing home delivery to them, a restaurant feature must be made available to them in the Essentials section.  

I would like to add this feature on a new page which will include the details of the restaurant ( such as an address, contact number, and image). Basically this page will help people to know about the restaurants available for them in this bad time. This page will only display the essential information about the restaurant.

I have also thought about sorting the restaurants on the basis of their location and also to include a search filter that will help the people to quickly find the restaurant of their choice.

Basically this feature will not only provide the restaurants a platform to show themselves but it will also help the jobless people to get the job of a delivery person as well. Since due to this COVID-19 situation it has become difficult for the restaurants to maintain themselves on platforms like swiggy and zomato, and also many small restaurants that are not capable of displaying them there can showcase themselves here.

**Comment by bee-rickey** [comment]
How do you collate this data? I presume it's quite a lot of data that needs to be collected? Also, wouldn't Zomato/Swiggy already have this?

**Comment by rohit154** [comment]
If we use google forms to collect data directly from the restaurant owners them it would be easy to collate this data. We will only be collecting the data such as 
-restaurant name
-complete address
-contact number
Now considering the platforms Swiggy and Zomato,swiggy charges around 18-23% per order for the restaurants on its platform similarly zomato charges around 1000rs. Looking at the current scenario all the restaurants are not in the position to pay this much of charge. Also, many people have lost their jobs in this meantime so these restaurants can also provide such people with a job of a delivery person.

**Comment by bee-rickey** [comment]
> If we use google forms to collect data directly from the restaurant owners them it would be easy to collate this data. We will only be collecting the data such as
> -restaurant name
> -complete address
> -contact number
> Now considering the platforms Swiggy and Zomato,swiggy charges around 18-23% per order for the restaurants on its platform similarly zomato charges around 1000rs. Looking at the current scenario all the restaurants are not in the position to pay this much of charge. Also, many people have lost their jobs in this meantime so these restaurants can also provide such people with a job of a delivery person.

Interesting. @sudevschiz @JunaidBabu thoughts?
